src/backend/joanie/core/serializers/admin.py

```diff
@@ -636,6 +636,30 @@ class Meta:
         read_only_fields = ["id", "course", "course_runs", "is_graded", "position"]
 
 
+class AdminOrderGroupSerializer(serializers.ModelSerializer):
+    """
+    Admin Serializer for OrderGroup model
+    """
+
+    nb_available_seats = serializers.SerializerMethodField(read_only=True)
+
+    class Meta:
+        model = models.OrderGroup
+        fields = [
+            "id",
+            "nb_seats",
+            "product",
+            "is_active",
+            "nb_available_seats",
+            "created_on",
+        ]
+        read_only_fields = ["id"]
```

Comment:
`product` should not be `read_only` too ?

Reply:
Same for 'created_on'